### DISCUSSION TITLE
Problem with Twilio Dependency

### DISCUSSION THREAD

**Discussion by Niswendel** [jef/streetmerchant#2055] [Help]
npm WARN @cliqz/adblocker-puppeteer@1.19.0 requires a peer of puppeteer@5.x but none is installed. You must install peer dependencies yourself.
npm WARN twilio@3.57.0 requires a peer of @types/express@^4.17.7 but none is installed. You must install peer dependencies yourself.
npm WARN twilio@3.57.0 requires a peer of @types/qs@6.9.4 but none is installed. You must install peer dependencies yourself.

Im having this problem. I have Git and Node installed, am i missing something?

**Comment by Mars-2030** [comment]
have you solved this?

**Comment by Mr-Tech-13** [comment]
Try npm update
